src/controllers/itemDetails/index.js

```diff
@@ -41,6 +41,7 @@ import 'elements/emby-scroller/emby-scroller';
 import 'elements/emby-select/emby-select';
 
 import 'styles/scrollstyles.scss';
+import '../../styles/lyrics.scss';
```

Comment:
I believe we can now use the absolute import path.
```suggestion
import 'styles/lyrics.scss';
import 'styles/scrollstyles.scss';
```

Reply:
Thanks! Styles aren't currently being used in itemDetails so this import has been removed (I forgot to remove it when I changed it)

Fixed [17c61f4](https://github.com/jellyfin/jellyfin-web/pull/4733/commits/17c61f4b45928a188276c88a0cc05b9d3a94e23a)